### PR TITLE
Tiny cleanup #3

### PR DESCRIPTION
**Pull request by krinart**
1. Simplify RequestId
2. Simplify InstanceId
3. Simplify operationFailedWriter
4. Rename RequestTrackingService.baseService -> RequestTrackingService.trackedMessagesReceive
5. Remove ".apply" for case classes while building AutoRequestIdFormat

### CODE REVIEW

File: akkeeper/src/main/scala/akkeeper/common/InstanceId.scala [hunk 1]
```diff
@@ -25,12 +25,11 @@ import spray.json._
   *                      belongs to.
   * @param uuid the unique ID of the instance.
   */
-case class InstanceId(containerName: String, uuid: UUID) {
+case class InstanceId(containerName: String, uuid: UUID = UUID.randomUUID()) {
```
[the remark this answers is not in the document]
Comment (krinart) [reply akkeeper/src/main/scala/akkeeper/common/InstanceId.scala]: Reverted

File: akkeeper/src/main/scala/akkeeper/common/RequestId.scala [hunk 2]
```diff
@@ -22,22 +22,16 @@ import spray.json._
   *
   * @param uuid the unique ID.
   */
-case class RequestId(uuid: UUID) {
+case class RequestId(uuid: UUID = UUID.randomUUID()) {
```
[the remark this answers is not in the document]
Comment (krinart) [reply akkeeper/src/main/scala/akkeeper/common/RequestId.scala]: Reverted

**Comment by izeigerman** [comment]
@krinart Oops, it seems like merging the previous PR caused conflicts for this one

**Comment by krinart** [comment]
Conflicts fixed.

**Pull request merged**